wix-data/wix-data/guides/Error Codes.md

```diff
@@ -33,18 +33,28 @@ When you use the Wix Data API, you may encounter errors. This list includes the
 |WDE0079| {your collection name} is a single item collection and item must have id {expected ID}, but got {received ID} instead.| The item in a single item collection has a fixed `_id` value. Check that your aren't trying to change this value in your code. You can find the item's `_id` value using the [Content Manager](https://support.wix.com/en/article/about-the-content-manager-7160473#collections) or by using [`find()`](/wix-data/wixdataquery/find) without any filters.
+|WDE0080| User request validation error. Incoming API request payload is invalid. | `collectionName` for a given data operation might not be provided. Check the payload data.
 |WDE0091| Database quota exceeded. Delete some data and try again.| Your site has exceeded the collection storage quota. [Learn more](https://support.wix.com/en/article/velo-about-data-quotas#collection-storage) about this quota and what you can do to resolve this issue.
 |WDE0092| Dataset is too large to sort.| The data you tried to sort didn't fit into memory. This usually occurs when large items are being sorted or there are too many items being sorted to fit into memory at once. [Add an index](https://support.wix.com/en/article/content-manager-adding-an-index-for-your-collection) to your collection to avoid using up memory.
 |WDE0093| Invalid filter for _publishStatus field, only .eq and .ne filters are allowed with 'DRAFT' and 'PUBLISHED' possible values.| When you enable the [Draft and Publish Items](https://support.wix.com/en/article/content-manager-using-the-draft-publish-items-feature) feature for a collection, a `_publishStatus` field is added to every item in that collection. If you want to filter a data request based on this field, you can only use the [`eq()`](/wix-data/wixdatafilter/eq) and [`ne()`](/wix-data/wixdatafilter/ne) functions. The only valid values for `_publishedStatus` are `'DRAFT'` and `'PUBLISHED'`. Check that your code isn't using other filter functions or values with `_publishedStatus`.
+|WDE0110| WD_BAD_REQUEST. | WixCodeNotEnabled - wix-data fails to resolve wix-code context. The error is not expected under normal circumstances, which usually means something is wrong with the system.
+|WDE0111| User request validation error. | `GridAppId` parameter must be provided in the SANDBOX segment.
 |WDE0115| WD_UNKNOWN_ERROR| Your external database requested an unsupported data operation.
 |WDE0116| WD_UNKNOWN_ERROR| Your external database responded to Wix with an error.
+|WDE0120| WD_BAD_REQUEST| Unsupported Driver Operation - `$collectionName` does not support `$operation` operation.
```

Comment:
Two questions:

1. Is this relevant for Velo users?
2. The "troubleshooting" reads like an error message. Are you sure it's not the error message itself?